src/AspNetCoreModuleV2/RequestHandler/inprocess/inprocesshandler.cpp

```diff
@@ -123,6 +124,24 @@ IN_PROCESS_HANDLER::SetAsyncCompletionStatus(
     m_requestNotificationStatus = requestNotificationStatus;
 }
 
+VOID
+IN_PROCESS_HANDLER::SetWebsocketSupportedVariable(
+    VOID
+)
+{
+    PCSTR pzWebsocketVar = NULL;
+    DWORD dwVersionLength;
+    HRESULT hr;
+
+    if (g_fWebSocketSupported == WebSocketStatus::WEBSOCKET_UNKNOWN)
```

Comment:
for naming conversion, g_XX means global variable. As you check websocket per application, it should use g_xxx

Reply:
I don't quite understand the difference you are specifying.